src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -955,7 +973,11 @@ private SafeProcessHandle GetProcessHandle(int access, bool throwIfExited)
                         }
                     }
                 }
-                return _processHandle;
+
+                // If we dispose of our contained handle we'll be in a bad state. NetFX dealt with this
+                // by doing a try..finally around every usage of GetProcessHandle and only disposed if
```

Comment:
Would it make senes to do the reverse for 2.0, though? I mean, we want to minimize the change. I'm not opposed, I just want to do the safest thing and leave cleanup for later.